backend/.env.example

```diff
@@ -1,5 +1,6 @@
 DJANGO_SETTINGS_MODULE=brainwave.settings.local
 CELERY_BROKER_URL=amqp://broker:5672//
+PYRIGHT_PYTHON_ENV_DIR=/home/user/app/.pyright
```

Comment:
This is where `pyright` is configured to save its environment in the project directory